spec/system/placements/organisations/users/user_views_other_users_spec.rb

```diff
@@ -67,6 +81,14 @@ def then_i_see_user_details(user:)
     expect(page).to have_content user.email
   end
 
+  def and_i_see_the_remove_user_link
+    expect(page).to have_link "Remove user"
+  end
+
+  def and_i_do_not_see_the_remove_user_link
+    expect(page).to_not have_link "Remove user"
+  end
```

Comment:
Oh! My mistake. I think you can probably ignore the comment above, since you've covered it in this system spec instead.